### ISSUE TITLE
Audiveris application doesn't load and run after new installation

### ISSUE THREAD

**Issue by Charvadam** [Audiveris/audiveris#582]
Dear Audiveris fellows, 

I'm trying to make the new downloaded Audiveris 5.2.5 application work with my new JDK 17.0.3.1 (64 bit), but I get into troubles.

'java -version' in a command prompt shows that I have a suitable Java Runtime Environment (JRE) : C:\Users\Administrateur>java -version -->
```
java version "17.0.3.1" 2022-04-22 LTS
Java(TM) SE Runtime Environment (build 17.0.3.1+2-LTS-6)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.3.1+2-LTS-6, mixed mode, sharing)
```
-> OK

(Paramètres > Paramètres associés > Paramètres avancés du système > Propriétés système > Paramètres système avancés > Variables d'environnement) : `JAVA_HOME = C:\Program Files\Java\jdk-17.0.3.1\bin`

Completing Audiveris 5.2.5 Setup
Audiveris 5.2.5 has been installed on your computer.

In C:\Programmes, a new folder appears, containing the bin and lib folders, an Audiveris shortcut and eventually an uninst application. Activating the Audiveris shortcut makes my browser display a 'https://github.com/Audiveris' tab.
In 'Program Files\Audiveris\bin, there is 
- a little file on 'Audiveris start up script for UN*X'
- an 'Audiveris' Windows command file, to launch the application
- an icon (icon-256)

**My question** : When I double-click on the Audiveris Windows command file, an empty frame shows up but disappears immediately. It is not possible to have the Audiveris application be loaded and run normally.
Does anybody have an idea on what's going wrong ?

I should be surprised hearing that I am the only one having this kind of installation trouble ...
Thanks in advance and kind regards.

**Comment by Charvadam** [comment]
Hello hbitteur ... and thank you for your advice.

Indeed, I have used the Audiveris Installer
and I have an Audiveris item in Windows launch menu which contains a Audiveris (icon), an uninstall item and a Website item.

I'm using Windows 10 (64) --> the Audiveris.bat file to launch it.
Unfortunately, when I run 'C:\Programmes\Audiveris\bin\Audiveris.bat', the shell replies that Audiveris.bat cannot be found. 
And when I run 'C:\Program Files\Audiveris\bin\Audiveris.bat', It thinks I want to call 'C:\Program' and doesn't recognize it as a valid command, of course.
See the snapshot. ![Command shell Audiveris 1](https://user-images.githubusercontent.com/106738301/174973214-15c4da41-a927-47a2-886e-127664e898ac.jpg)

Brrrrrr......
Kind regards and thanks

**Comment by hbitteur** [comment]
Instead of:
> C:\Program Files\Audiveris\bin\Audiveris.bat
use (mind the quotes to jump over the space character):
> "C:\Program Files"\Audiveris\bin\Audiveris.bat

**Comment by Charvadam** [comment]
Dear Hervé

Indeed, names with a space(s) ought to be written between quotes. I was forgotten; it's such a long time that I used DOS commands. 
See here, please :
![Command shell Audiveris 2](https://user-images.githubusercontent.com/106738301/175032824-c7e9b6d6-4861-4ddb-a8e2-d3f0a8c4b8a7.jpg)
I thought that 'the location of your Java installation' was to be found in the '\bin' folder.

I just managed to change the JAVA_HOME parameter into C:\Program Files\Java\jdk-17.0.3.1 - i.e. without '\bin' - but also without success.
Now, instead of opening a command shell that dissapears directly, it opens a new tag in my browser, calling 'audiveris.org' but without success to download a content. After a few seconds : 
> The page you are looking for is temporarily unavailable.
> Please try again later.
> 2cb0eab4b0fc7a57bf50802233c73e17 43f1c838bc4f70529ca3eef254b07f9f 3b9972dc4ed0b32e745bf68169777f51

--> without '\bin' is a wrong way.

Unfortunately, also not to find in '[JDK 17 Documentation - Home](https://docs.oracle.com/en/java/javase/17/index.html)' , i.e. in 'Manage & Troubleshoot'.
 
Thanks for your ideas.

**Comment by hbitteur** [comment]
Sorry, could you better explain what the result was finally? (after quoting the spaces and setting JAVA_HOME correctly)
I don't understand why a browser got involved.
Are you still trying from a terminal window?
If so, could you post the final window content?

**Comment by Charvadam** [comment]
Dear Hervé

Simply that when I changed the JAVA_HOME parameter from C:\Program Files\Java\jdk-17.0.3.1**\bin** to C:\Program Files\Java\jdk-17.0.3.1 in the 'Environment variables' window, and activate the icon of Audiveris (a mistake), this triggers the display in my browser of a not valid URL : audiveris.org instead of calling the Audiveris.bat file. 
Sorry that I forgot to call again the Audiveris.bat from within a terminal window (= the command shell) this time. That happens when you do several things together.
Now that I did it, I see that the Audiveris application is called : see following picture containing the terminal window and the main application frame of Audiveris : ![Terminal window + Application frame 1](https://user-images.githubusercontent.com/106738301/175053982-bc963162-5c81-4ea3-9789-53fa774c8288.jpg)

It seems that the application operates correctly : the main frame does appear and seems to work properly ! 
I'm discovering that the Audiveris-icon on the desktop calls the main website page 'http://www.audiveris.org' (which works OK now). This one just contains a comprehensive menu of the Audiveris facilities. 
So, Audiveris does not provide an icon that calls directly `"C:\Program Files"\Audiveris\bin\Audiveris.bat`. 

It seems that my mistake simply was that I put C:\Program Files\Java\jdk-17.0.3.1**\bin** instead of C:\Program Files\Java\jdk-17.0.3.1 into the JAVA_HOME variable.

Many thanks for your help.
